pkg/geo/geomfn/voronoi.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package geomfn
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/geo"
+	"github.com/cockroachdb/cockroach/pkg/geo/geos"
+)
+
+// Computes the Voronoi Polygons from the vertices of the supplied geometry.
+func VoronoiPolygons(
+	a geo.Geometry, tolerance float64, onlyEdges int,
```

Comment:
`onlyEdges` should be a bool, converted into a C int at the very last moment (i.e. convert this to an int in geos.go when calling the c library).